lib/daisy_ui_components/input.ex

```diff
@@ -151,6 +154,59 @@ defmodule DaisyUIComponents.Input do
     """
   end
 
+  def input(%{type: "autocomplete"} = assigns) do
+    selected_label =
+      Enum.find_value(assigns.options, fn {label, value} ->
+        if to_string(value) == to_string(assigns.value), do: label
+      end)
+
+    assigns =
+      assigns
+      |> assign(:selected, selected_label)
+      |> update(:on_query, fn
+        %JS{} = js -> js
+        event when is_binary(event) -> JS.push(event)
+      end)
+
+    ~H"""
+    <div class="dropdown">
+      <.input
+        type="text"
+        id={@id <> "_label"}
+        name={@id <> "_label"}
+        class={@class}
+        color={@color}
+        phx-change={@on_query}
+        phx-debounce={300}
+        value={@selected}
+        autocomplete="off"
+        placeholder={@rest[:placeholder]}
+        tabindex="0"
+      />
+      <ul
+        tabindex="1"
+        class="menu dropdown-content bg-base-100 rounded-box z-1 max-h-80 p-2 w-full shadow flex-nowrap overflow-auto"
+      >
+        <li :for={{label, value} <- @options}>
+          <button
+            type="button"
+            class={to_string(value) == to_string(@value) && "menu-active"}
```

Comment:
if false I thnk this will render the class `false` in html, if wrap up as list it will not display the class
```suggestion
            class={[to_string(value) == to_string(@value) && "menu-active"]}
```